src/app/clusters/color-control-server/color-control-server.cpp

```diff
@@ -223,7 +199,7 @@ class DefaultColorControlSceneHandler : public scenes::DefaultSceneHandlerImpl
 #endif
 
         // Initialize action attributes to default values in case they are not in the scene
-        uint8_t targetColorMode    = 0x00;
+        auto targetColorMode       = EnhancedColorMode::kCurrentHueAndCurrentSaturation;
```

Comment:
```suggestion
        auto targetColorMode       = EnhancedColorModeEnum::kCurrentHueAndCurrentSaturation;
```
since this is new, why not just use the right name?